### PR TITLE
fixed bug rotation in sprite pacman

### PR DESCRIPTION
**Pull request by Laura4lilavati**
fixed bug rotation sprite pacman.

### CODE REVIEW

**Comment by maryangela** [comment]
Your branch has conflicts with 3 classes, please check

**Comment by Laura4lilavati** [comment]
> Your branch has conflicts with 3 classes, please check

Thank you, the conflicts are already fixed

**Comment by Laura4lilavati** [comment]
Fixed rhe movements of pacman in the board, it goes straight if can't go sideways by keyboard.

**Comment by julia-jpg** [comment]
everything ok

File: src/main/java/pacman/ActionKeyPacman.java [hunk 1]
```diff
@@ -7,8 +7,14 @@
 public class ActionKeyPacman implements KeyListener {
 
     private DrawComponents drawComponents;
-    private int key;
     public static final int TIMER_PACMAN = 200;
+    public static final String MOVE_LEFT = "left";
+    public static final String MOVE_RIGHT = "right";
+    public static final String MOVE_UP = "up";
+    public static final String MOVE_DOWN = "down";
```
Comment (roht0515) [inline src/main/java/pacman/ActionKeyPacman.java:14]: are these constants used somewhere? i checked but i didn't find its using in this class.

File: src/main/java/pacman/ActionKeyPacman.java [hunk 2]
```diff
@@ -54,4 +62,20 @@ public void keyReleased(final KeyEvent event) {
...
+     */
+    public String assignMovement(final int keyValue) {
+        if (keyValue == KeyEvent.VK_RIGHT) {
+            posToMove = "right";
+        } else if (keyValue == KeyEvent.VK_LEFT) {
+            posToMove = "left";
+        } else if (keyValue == KeyEvent.VK_DOWN) {
+            posToMove = "down";
+        } else if (keyValue == KeyEvent.VK_UP) {
+            posToMove = "up";
```
Comment (roht0515) [inline src/main/java/pacman/ActionKeyPacman.java]: you could use the constans in this part. example 
"posToMove = MOVE_RIGHT"
the same with the others

File: src/main/java/pacman/ActionKeyPacman.java [hunk 3]
```diff
@@ -7,8 +7,14 @@
...
 
     private DrawComponents drawComponents;
-    private int key;
     public static final int TIMER_PACMAN = 200;
+    public static final String MOVE_LEFT = "left";
+    public static final String MOVE_RIGHT = "right";
+    public static final String MOVE_UP = "up";
+    public static final String MOVE_DOWN = "down";
+    private int key, preKey;
+    private Boolean aux = true;
```
Comment (maryangela) [inline src/main/java/pacman/ActionKeyPacman.java]: maybe you could change the name of the "aux" variable to a more descriptive one

File: src/main/java/pacman/DrawComponents.java [hunk 4]
```diff
@@ -29,6 +30,11 @@
     private Creator creator;
     private Timer timer;
     private int seconds = 0;
+    public static final int CASE_LEFT = 0;
+    public static final int CASE_RIGHT = 1;
+    public static final int CASE_UP = 2;
+    public static final int CASE_DOWN = 3;
```
Comment (roht0515) [inline src/main/java/pacman/DrawComponents.java]: If you are not using these constants, please delete them

File: src/main/java/pacman/ActionKeyPacman.java (hunk 1 above)
Comment (Laura4lilavati) [reply src/main/java/pacman/ActionKeyPacman.java:14]: thank you for your comment, hey are used in the last commit.

File: src/main/java/pacman/ActionKeyPacman.java (hunk 2 above)
Comment (Laura4lilavati) [reply src/main/java/pacman/ActionKeyPacman.java]: thank you, it's fixed in the last commit

File: src/main/java/pacman/DrawComponents.java (hunk 4 above)
Comment (Laura4lilavati) [reply src/main/java/pacman/DrawComponents.java]: thanks, they are removed.

File: src/main/java/pacman/ActionKeyPacman.java (hunk 3 above)
Comment (Laura4lilavati) [reply src/main/java/pacman/ActionKeyPacman.java]: thank you, it's changed in the last commit

**Pull request merged**